src/main/java/use_case/getpost/GetPostDataAccessInterface.java

```diff
@@ -0,0 +1,27 @@
+package use_case.getpost;
+
+import java.util.List;
+
+import entity.Post;
```

Comment:
Think it might be an issue importing an entity into the DAO layer....

Reply:
Yea tbh I'm not sure if we should be accessing entities in DAOs (clean architecture), I did that in my user dao, which I return a user object with the data retrieved from the database. now I'm thinking of just returning a JSONObject, and let the use cases convert them. We'll see tmrw

Reply:
Yeah I agree! We can confirm with the TA